lib/pf/Switch.pm

```diff
@@ -2726,40 +2727,60 @@ Default implementation.
 =cut
 
 sub returnRadiusAccessAccept {
-    my ($self, $vlan, $mac, $port, $connection_type, $user_name, $ssid, $wasInline, $user_role) = @_;
+    my ($self, $args) = @_;
     my $logger = $self->logger();
 
-    # Inline Vs. VLAN enforcement
+    my $filter = pf::access_filter::radius->new;
+    my $rule = $filter->test('returnRadiusAccessAccept', $args);
     my $radius_reply_ref = {};
+    $radius_reply_ref = $filter->handleAnswerInRule($rule,$args);
```

Comment:
Shouldn't that process happen after ?

It would be nice to be able to manipulate what the 'normal' flow sent us and modify it or just rewrite from scratch